### PR TITLE
Lps 105105 entity error

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js [hunk 1]
```diff
@@ -410,6 +439,7 @@ class CriteriaRow extends Component {
 				{error ? (
 					<ClayButton
 						className="btn-outline-danger btn-sm"
+						displayType=""
```
Comment (dgarciasarai) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js:452]: 🤔

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js (hunk 1 above)
Comment (andresfulla) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js:452]: this button can be improved. Trouble is: `displayType`defaults to `primary` and overwrites `btn-outline-danger`. I'm sad 🙁

Comment (andresfulla) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js]: 🙁

**Pull request closed without merging**